draft-ietf-httpbis-message-signatures.md

```diff
@@ -357,6 +366,55 @@ The following example shows canonicalized values for different component identif
 
 Note that the value for `key="c"` has been re-serialized according to the strict `member_value` algorithm.
 
+
+### Binary-wrapped HTTP Fields {#http-header-byte-sequence}
+
+If the value of the the HTTP field in question is known by the application to cause problems with serialization, particularly with combination of multiple values as discussed in {{security-non-list}}, the signer MAY include the `bs` parameter in a component identifier to indicate the values of the fields need to be wrapped as binary structures before being combined.
```

Comment:
If we are aware of other common fields that need to be handled this way, let's list them in {{security-non-list}} as well.

Reply:
Set-Cookie is the only common one that I found that fits this, but I didn't want the language to preclude other weirdness out there.